.reflame.config.jsonc

```diff
@@ -45,19 +45,19 @@
 			"HEIGHT_CLIENT_ID": "w8GzB4GI8gGUE2iEAQMY8Wck44gK2ek2i4Q8q4myKQY",
 			"DEMO_SESSION_URL": "/1/sessions/SGKFqG4mE8au2UmHjT0BhjO5Q15f",
 			"LINEAR_CLIENT_ID": "f60ff43c7376d0aceaa1e111db39e60d",
-			"DEMO_PROJECT_ID": "1344",
+			"DEMO_PROJECT_ID": "1344"
```

Comment:
@Vadman97 @ccschmitz what should I do here?

Reply:
I get these too and wasn't sure why. I've just been committing them as I don't think they really impact anything. Maybe a conflict with our editor settings and someone else's.

Reply:
You need the trailing comma. Run `yarn format:all` to fix. 

We might need to tune something in reflame that's injecting these (or maybe it's from vscode). @lewisl9029 - any ideas?

Reply:
thanks.

Reply:
Ah yep, that's my bad, I forgot to update the env script after https://github.com/highlight/highlight/pull/5571

Thanks for the ping @et, will put up a PR to fix asap!

Reply:
Here's the PR: https://github.com/highlight/highlight/pull/5641